README.rst

```diff
@@ -208,6 +208,21 @@ To run specific tests, you can specify the test name too, something like:
 
 The ``--nocapture`` argument can be useful to see some output that otherwise is captured by nosetests.
 
+Also, you can run tests against an instance of Artifactory. Those tests should add the attribute
```

Comment:
Maybe it would be good to actually add a command line example like ``nosetests . -A "artifactory_ready"``?